RENOVATE.md

```diff
@@ -147,6 +147,7 @@ alerts, enabled by a previous preset, will use a `rangeStrategy` of `update-lock
     {"matchPackagePatterns": ["^mezzio/"], "groupSlug": "mezzio", "groupName": "all Mezzio packages"},
     {
         "matchPackageNames": ["php"],
+        "allowedVersions": ">=8.0"
```

Comment:
OK, but this isn't documented in our stuff :D 

Also, should this be `^8.0`? :thinking:

Reply:
The current description fits for this aswell, or I can break down exactly how this bit works instead if desired.

^8.0 works too, assuming this is updated each time, it shouldn't make a difference. I'll update though.